petab_select/candidate_space.py

```diff
@@ -115,6 +107,10 @@ def __init__(
         if self.previous_predecessor_model is None:
             self.previous_predecessor_model = self.predecessor_model
 
+        self.method = method
+        if self.method is None:
+            self.method = self.governing_method
```

Comment:
```suggestion
            self.method = CandidateSpace.governing_method
```
Make it explicit that we're using a class attribute.